### PR TITLE
Use proper dictionary for new calendar event notifications data

### LINKED ISSUE #3805: Event title Notification
http://www.simplemachines.org/community/index.php?topic=551078.0

### PR DESCRIPTION
**Pull request by colinschoen**
Fixes #3805

Signed-off-by: Colin Schoen <cschoen@berkeley.edu>

### CODE REVIEW

File: Sources/tasks/EventNew-Notify.php [hunk 1]
```diff
@@ -80,7 +80,12 @@ public function execute()
 					'content_id' => $this->_details['event_id'],
 					'content_action' => empty($this->_details['sender_id']) ? 'new_guest' : 'new',
 					'is_read' => 0,
-					'extra' => json_encode(array($this->_details['event_id'], $this->_details['event_title'])),
+					'extra' => json_encode(
+					    array(
+					        "event_id" => $this->_details['event_id'],
+                            "event_title" => $this->_details['event_title']
```
Comment (live627) [inline Sources/tasks/EventNew-Notify.php]: spacing...

**Review by live627 — CHANGES_REQUESTED** [review]

**Comment by colinschoen** [comment]
Should be fixed. I need to sort out my PHPStorm vim plugin indentation settings.

**Review by live627 — APPROVED** [review]

**Comment by colinschoen** [comment]
I don't understand this lol...

screen shot 2017-01-06 at 17 21 01

**Comment by colinschoen** [comment]
The indentation is still showing as screwed up on github though even VIM and PHPStorm are showing it as correct...

**Comment by live627** [comment]
eh salright I can fix it later....

**Pull request merged**